backend/Dockerfile

```diff
@@ -9,13 +9,10 @@ RUN pip install --upgrade -r /ORAssistant-backend/requirements.txt
 COPY . .
 
 # Prereqs
-RUN apt-get update && apt-get install -y pandoc git wget
```

Comment:
Pandoc is no longer needed?

Reply:
Its not needed in the docker image, as we can pull manpages directly from Huggingface